vms/event/services.py

```diff
@@ -163,3 +166,11 @@ def remove_empty_events_for_volunteer(event_list, volunteer_id):
         if job_list:
             new_event_list.append(event)
     return new_event_list
+
+def remove_invalid_events(event_list):
+    """Removes all the events from an event list which have been occured already(expired)"""
+    new_event_list = []
+    for event in event_list:
+        if date.today() <= event.end_date:
```

Comment:
please store date.today() in a variable above, or this gets recalculated on each iteration.